bin/run1_cpu_generate_rf.py

```diff
@@ -0,0 +1,90 @@
+"""
```

Comment:
In order for this to be a standalone executable, the first line of this script should be, e.g., 
```
#!/usr/bin/env python
```